composer.json

```diff
@@ -9,24 +9,22 @@
     "license": "MIT",
     "require": {
         "php": "^8.0.2",
-        "opcodesio/log-viewer": "^1.2",
-        "fruitcake/laravel-cors": "^2.0",
+        "opcodesio/log-viewer": "^1.9",
         "guzzlehttp/guzzle": "^6.0|^7.0",
```

Comment:
Can we upgrade to `7.2` only?
https://github.com/laravel/laravel/blob/10.x/composer.json#L9